graylog2-server/pom.xml

```diff
@@ -564,6 +564,11 @@
             <groupId>com.maxmind.geoip2</groupId>
             <artifactId>geoip2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.ipinfo</groupId>
+            <artifactId>ipinfo-api</artifactId>
+            <version>${ipinfo.version}</version>
+        </dependency>
```

Comment:
@roberto-graylog Is this dependency required? It seems to be unused.

Reply:
It is not.  Thanks for the catch.  I forgot that I switched to the existing code that uses the MaxMind Db Reader.  I have removed the dependency.